diff -ru hdf5-1.8.23/config/cmake_ext_mod/ConfigureChecks.cmake hdf5-1.8.23-patched/config/cmake_ext_mod/ConfigureChecks.cmake
--- hdf5-1.8.23/config/cmake_ext_mod/ConfigureChecks.cmake	2023-02-07 15:36:26.000000000 +0100
+++ hdf5-1.8.23-patched/config/cmake_ext_mod/ConfigureChecks.cmake	2025-07-30 23:05:54.152300022 +0200
@@ -529,7 +529,7 @@
 
 CHECK_FUNCTION_EXISTS (tmpfile           ${HDF_PREFIX}_HAVE_TMPFILE)
 CHECK_FUNCTION_EXISTS (asprintf          ${HDF_PREFIX}_HAVE_ASPRINTF)
-CHECK_FUNCTION_EXISTS (vasprintf         ${HDF_PREFIX}_HAVE_VASPRINTF)
+CHECK_FUNCTION_EXISTS (vxxxsprintf         ${HDF_PREFIX}_HAVE_VASPRINTF)
 CHECK_FUNCTION_EXISTS (waitpid           ${HDF_PREFIX}_HAVE_WAITPID)
 
 CHECK_FUNCTION_EXISTS (vsnprintf         ${HDF_PREFIX}_HAVE_VSNPRINTF)
